### PR TITLE
Separate raw kv user documents from BR

### PR DESCRIPTION
**Pull request by IANTHEREAL**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
rawkv is a separate product, so let's separate raw kv user documents from BR. 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by IANTHEREAL** [comment]
@Liuxiaozhen12 @kennytm @3pointer  PTAL

File: br/rawkv-backup-and-restore.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    --cf default
+```
+
+以上命令会备份 default CF 上 `[0x31, 0x3130303030303030)` 之间的所有键到 `$BACKUP_DIR` 去。
+
+这里，`--start` 和 `--end` 的参数会先依照 `--format` 指定的方式解码，再被送到 TiKV 上去，目前支持以下解码方式：
+
+- "raw"：不进行任何操作，将输入的字符串直接编码为二进制格式的键。
+- "hex"：将输入的字符串视作十六进制数字。这是默认的编码方式。
+- "escape"：对输入的字符串进行转义之后，再编码为二进制格式。
```
Comment (kennytm) [inline br/rawkv-backup-and-restore.md:36]:
```suggestion
- "escaped"：对输入的字符串进行转义之后，再编码为二进制格式。
```

we should also explain it is *backslash*-escaped, in the form like `'abc\xFF\x00\r\n'`.

**Review by kennytm — COMMENTED** [review]
rest lgtm

File: br/rawkv-backup-and-restore.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+
+## Raw KV 恢复（实验性功能）
+
+和 [Raw KV 备份](#raw-kv-备份实验性功能)相似地，恢复 Raw KV 的命令如下：
```
Comment (hfxsd) [inline br/rawkv-backup-and-restore.md:41]:
```suggestion
和 [Raw KV 备份](#raw-kv-备份实验性功能)相似，恢复 Raw KV 的命令如下：
```

**Comment by IANTHEREAL** [comment]
I want to close this pr, because https://github.com/pingcap/docs-cn/pull/8273 also work on it

**Pull request closed without merging**